lib/frame10.rb

```diff
@@ -0,0 +1,51 @@
+class Frame10
+
+  def initialize
+    @roll_number = 0
+    @total_rolls = 2
+  end
+
+  def strike(roll)
+    if roll == 10
+      puts "You got a strike!"
+      return true
+    end
+  end
+
+  def spare
+    if ((@roll1 + @roll2) == 10)
+      puts "You got a spare!"
+      return true
+    end
+  end
```

Comment:
Frame10 seems to be a type of Frame - could use inheritance or composition perhaps to inherit strike and spare methods?

Reply:
and roll1 and roll2